cs/Platforms/Client.Wpf/Client.Wpf.csproj

```
@@ -1,11 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net471</TargetFramework>
+    <TargetFramework>net471;net6.0-windows</TargetFramework>
```

This needs to be changed to `<TargetFrameworks>` to use multi-targeting:

```suggestion
    <TargetFrameworks>net471;net6.0-windows</TargetFrameworks>
```